locales-pending/premium.ftl

```diff
@@ -256,3 +256,55 @@ false-door-test-content-part-two = Automatically remove data from sites trying t
 false-door-test-content-part-two-dashboard = Auto-delete data from sites trying to sell it.
 false-door-test-cta = Count me in
 false-door-test-popup-close = Close
+
+# High Risk Data Breaches
+
+high-risk-breach-heading = Here’s what to do
+high-risk-breach-subheading = This requires access to your sensitive info, so you’ll need to manually fix it.
+# Variables
+# $num_breaches is the number of breaches where the high risk data was found.
+high-risk-breach-summary = { $num_breaches ->
+  [one] It appeared in { $num_breaches } data breach:
+  *[other] It appeared in { $num_breaches } data breaches:
+}
+# Variables
+# $breach_name is the name of the breach where the high risk data was found.
+# $breach_date is the date when the breach occurred. 
+# An example of this string is Twitter on 13/09/18.
+high-risk-breach-name-and-date = { $breach_name } <breach_date>on { $breach_date }</breach_date>
+high-risk-breach-mark-as-fixed = Mark as fixed
+high-risk-breach-skip = Skip for now
+# Variables:
+# $estimated_time is the estimated time it would take for a user to complete breach resolution steps. It is never singular.
+high-risk-breach-estimated-time = Your estimated time: { $estimated_time }+ minutes
+
+# Credit Card Breaches
+
+high-risk-breach-credit-card-title = Your credit card number was exposed.
+high-risk-breach-credit-card-description = Anyone who gets it can make unauthorized purchases that you may be liable for. Act now to prevent financial harm.
+high-risk-breach-credit-card-step-one = If you still have this card, contact the issuer to report it stolen.
+high-risk-breach-credit-card-step-two = Request a new card with a new number.
+high-risk-breach-credit-card-step-three = Check your accounts for unauthorized charges.
+
+# Bank Account Breaches
+
+high-risk-breach-bank-account-title = Bank Account Data Breach
+high-risk-breach-bank-account-description = Taking action ASAP could give you more legal protections to help you recover any losses.
+high-risk-breach-bank-account-step-one = Notify your bank immediately that your account number has been compromised.
+high-risk-breach-bank-account-step-two = Change your account number.
+high-risk-breach-bank-account-step-three = Check your accounts for unauthorized charges.
+
+# Social Security Number Breaches
+
+high-risk-breach-social-security-title = Social Security Number Data Breach
+high-risk-breach-social-security-description = Scammers can open up new loans or credit cards with your social security number. Act fast to prevent financial harm.
```

Comment:
L284 says:
> "Act _now_ to prevent financial harm."

This says:
> "Act _fast_ to prevent financial harm."

Any interesting reason for the difference in wording; or should we stick with one variant for consistency?